frontend/src/components/eMIB/Inbox.jsx

```diff
@@ -41,6 +41,7 @@ class Inbox extends Component {
   changeEmail = index => {
     this.props.readEmail(this.state.currentEmail);
     this.setState({ currentEmail: index });
+    this.props.readEmail(index);
```

Comment:
I think we should test changing an email with a simulated click, calls the readEmail prop function using a mock/spy.

Reply:
Added in commit #https://github.com/code-for-canada/project-thundercat/pull/203/commits/27a95cc9693a51f92351ed918fa94fe8ff04cff9